sources/osg/Matrix.js

```diff
@@ -611,6 +649,22 @@ var Matrix = {
         };
     } )(),
 
+    preMultRotate: function ( matrix, quat ) {
+        if ( Quat.zeroRotation( quat ) )
+            return;
+        var r = Matrix.create();
```

Comment:
it's slow to create object like that, why not using regular function directly if it's slow ?